### PR TITLE
Signed Fields Docs

### PR DESCRIPTION
**Pull request by kpollich**
Initial Signed Fields documentation. 

Signed fields are the attributes needed to interface with CyberSource. I've opened this PR to start discussion on how we want to document and implement the endpoint to retrieve this set of attributes. 

There is no `signed-fields` data model, these are derived fields based on the current user and cart.

### CODE REVIEW

File: signed-fields.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+These are the properties required to POST a potential transaction to CyberSource for validation. These fields are derived primarily from the currently authenticated user as well as the user's cart.
+
+## Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| transaction_type | string | CyberSource methods being used. Hard coded as "sale,create_payment_token" |
+| payment_method | string | How the transaction is being paid. Hard coded as "card" |
+| update_payment_token | boolean | ? |
```
Comment (kpollich) [inline signed-fields.md]: @pperera9 @Justin17584 - What does this `update_payment_method` field mean for us? I couldn't figure it out from existing code, etc. Do we support using existing tokens to verify new carts? Need some clarification on this process.

File: signed-fields.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Signed Fields
+
+These are the properties required to POST a potential transaction to CyberSource for validation. These fields are derived primarily from the currently authenticated user as well as the user's cart.
+
+## Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| transaction_type | string | CyberSource methods being used. Hard coded as "sale,create_payment_token" |
```
Comment (scottgonzalez) [inline signed-fields.md]: s/methods/method/

**Comment by scottgonzalez** [comment]
The routes aren't defined.

**Comment by scottgonzalez** [comment]
This contains a fair amount of internal documentation.

File: signed-fields.md (hunk 1 above)
Comment (Justin17584) [inline signed-fields.md]: allow_payment_token_update
Indicates whether the customer can update the billing, shipping, and payment information on the order review page. This field can contain one of the following values:
■
true: customer can update details.
■
false: customer cannot update details.
update_payment_token (R)
Enumerated String
String (5)

http://apps.cybersource.com/library/documentation/dev_guides/Secure_Acceptance_SOP/html/wwhelp/wwhimpl/js/html/wwhelp.htm#href=app_fields_SOP.html

**Comment by kpollich** [comment]
Closing for now to clarify some use cases, etc.

**Pull request closed without merging**